src/components/SearchResults.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import "../styles/searchResults.css";
+
+const SearchResults = ({results}) => {
+
+  if (!results.length) {
+    return (
+      <div className="no-results">
+    <p>No results</p>
+    </div>
+    )
+  } else {
+    return (
+      <>
+      <div>
+        {results.map((result) => (
+          <img className="search-results" src={result} alt="spaceImage" />
+        ))}
```

Comment:
you're getting a warning in the browser console when you search for something, because you haven't provided a key prop to this element. When doing a map in react, you should provide a unique key to your elements that is unique to where the element came from but doesn't care about it's position in the array (i.e. do not use the index as the key). You can read more about the key prop [here](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key). Because the `result` is a unique link to an image, you can use that as the key prop in this example, e.g.
```
{results.map((result) => (
  <img
    className="search-results"
    src={result}
    alt="spaceImage" 
    key={result}
  />
))}
```